doc/userguide/rule-management/rule-reload.rst

```diff
@@ -1,32 +1,83 @@
 Rule Reloads
 ============
 
-Suricata can reload the rules without restarting. This way, there
-is minimal service disruption.
+Suricata was designed to reload rules while it is actively processing
+network traffic to minimize service disruption.
 
-This works by sending Suricata a signal or by using the unix socket. When Suricata is told to reload the rules these are the basic steps it takes:
+Suricata must be administratively directed to reload rules while it is running; there
+are two ways to cause this:
 
-* Load new config to update rule variables and values.
-* Load new rules
-* Construct new detection engine
-* Swap old and new detection engines
-* Make sure all threads are updated
-* Free old detection engine
+* Via a UNIX domain socket with the command ``reload-rules`` or ``ruleset-reload-nonblocking``
+* Upon receipt of the ``USR2`` signal
 
-Suricata will continue to process packets normally during this process. Keep in mind though, that the system should have enough memory for both detection engines.
-
-Signal::
+Reload Triggers
+~~~~~~~~~~~~~~~
+There are multiple ways to trigger a rule reload. Note that the steps using ``suricatasc``
+can be done using the suricata`
+Via process signal
+^^^^^^^^^^^^^^^^^^
 
   kill -USR2 $(pidof suricata)
 
-There are two methods available when using the Unix socket.
+Via the UNIX domain socket.
 
-Blocking reload ::
+Blocking reload
```

Comment:
What is being blocked ?

Reply:
The caller that is initiating the reload. I'll clarify in the doc.